src/frontend/ts/state/chessStore.ts

```diff
@@ -240,9 +185,6 @@ const useChessStore = create<ChessState>((set, get) => ({
         //
         //  const displayGame = get().displayGame
         //  if ((displayGame !== null) && (gameID === displayGame.gameID)) {
-        //    //  XX: I don't want to set the DisplayGame to null, I want it
-        //    //      to be the last move the just archived game.
-        //    //
         //    get().setDisplayGame(null)
```

Comment:
a little experiment for not having a game-over, immediately boot to `practiceBoard`.